internal/pkg/otel/configtranslate/otelconfig.go

```diff
@@ -261,13 +264,13 @@ func getExporterTypeForComponent(comp *component.Component) (otelcomponent.Type,
 }
 
 // unitToExporterConfig translates a component.Unit to an otel exporter configuration.
-func unitToExporterConfig(unit component.Unit, exporterType otelcomponent.Type, inputType string) (map[string]any, error) {
+func unitToExporterConfig(unit component.Unit, exporterType otelcomponent.Type, inputType string) (exportersCfg map[string]any, queueSettings map[string]any, err error) {
```

Comment:
The docstring for this function should be updated to include the new return value.